.travis.yml

```diff
@@ -16,14 +16,41 @@ jobs:
   - python: 3.9-dev
   - <<: *latest_py3
     env: TOXENV=docs
+    #PPC64LE
+  - arch: ppc64le
+    python: pypy3
+  - arch: ppc64le
+    python: 3.5
+  - &latest_py3_ppc
+    arch: ppc64le
+    python: 3.8
+  - <<: *latest_py3_ppc
+    env: LANG=C
+  - arch: ppc64le
+    python: 3.9-dev
+  - <<: *latest_py3_ppc
+    env: TOXENV=docs
```

Comment:
Is it really necessary to build the docs on PPC?

Reply:
Nope. Removed.